src/main/java/lk/AVSEC/Welfare/asset/employee/controller/EmployeeController.java

```diff
@@ -105,7 +105,7 @@ public String employeePage(Model model) {
         }
         /*  Employee employee = employeeService.findById(id);*/
         model.addAttribute("employees", employees);
-        model.addAttribute("contendHeader", "Employee");
+        model.addAttribute("contendHeader", "Employee List");
```

Comment:
1. model.addAttribute("contendHeader", "Employee List");
if change following is most suitable header
contendHeader = All our employee